StockServiceFactory/src/main/java/edu/mbrooks/advancedjava/StockFactory/StockFactory.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Program Name:    StockFactory.java
+ * Course:          Advanced Java Programming, Summer 2018
+ * Copyright:       (C) 2018
+ */
+
+/**
+ * @author Michael Brooks
+ * @version 1.0
+ */
+package edu.mbrooks.advancedjava.StockFactory;
+import edu.mbrooks.advancedjava.StockQuote.*;
+import edu.mbrooks.advancedjava.StockFactory.*;
+import edu.mbrooks.advancedjava.StockService.*;
+
+/**
+ *  The StockTrader class implements the SockService interface
+ */
+class StockTrader implements StockService {
```

Comment:
this is not quite the right approach, but we talked about this in chat, so I think you are OK now.

Reply:
Yup, renamed the class in assignment 3